messages/run.js

```diff
@@ -1,101 +1,100 @@
-		'eslintConfigDescription': 'location of eslintrc config to customize eslint engine',
-		'eslintConfigDescriptionLong': 'Location of eslintrc to customize eslint engine',
-		'pmdConfigDescription': 'location of PMD rule reference XML file to customize rule selection',
-		'pmdConfigDescriptionLong': 'Location of PMD rule reference XML file to customize rule selection',
-		"verboseViolationsDescription": "retire-js violation messages include more details",
-        "verboseViolationsDescriptionLong": "retire-js violation messages contain details about each vulnerability (e.g. summary, CVE, urls, etc.)"
-
+		"stDescription": "throw an error when a violation threshold is reached, the --normalize-severity is invoked, and severity levels are reset to the baseline",
+        "stDescriptionLong": "Throws an error when violations are found with equal or greater severity than the provided value. --normalize-severity is invoked and severity levels are reset to the baseline. Normalized severity values are: 1 (high), 2 (moderate), and 3 (low). Exit code is the most severe violation.",
+		"nsDescription": "return normalized severity 1 (high), 2 (moderate), and 3 (low), and the engine-specific severity",
+        "nsDescriptionLong": "Returns normalized severity 1 (high), 2 (moderate), and 3 (low), and the engine specific severity. For the html option, the normalized severity is displayed instead of the engine severity.",
+		'engineDescription': "specify which engines to run",
+		'engineDescriptionLong': "Specifies one or more engines to run. Submit multiple values as a comma-separated list.",
+		'eslintConfigDescription': 'specify the location of eslintrc config to customize eslint engine',
+		'eslintConfigDescriptionLong': 'Specifies the location of eslintrc config to customize eslint engine.',
+		'pmdConfigDescription': 'specify location of PMD rule reference XML file to customize rule selection',
+		'pmdConfigDescriptionLong': 'Specifies the location of PMD rule reference XML file to customize rule selection.',
+		"verboseViolationsDescription": "return retire-js violation message details",
+        "verboseViolationsDescriptionLong": "Returns retire-js violation messages details about each vulnerability, including summary, Common Vulnerabilities and Exposures (CVE), and URLs."
 	},
 	"validations": {
-		"methodLevelTargetingDisallowed": "Target '%s' is invalid, as this command does not support method-level targeting",
-		"outfileFormatMismatch": "Your chosen format %s does not appear to match your output file type of %s.",
+		"methodLevelTargetingDisallowed": "The target '%s' is invalid because method-level targeting isn't supported with this command.",
+		"outfileFormatMismatch": "The selected output format doesn't match the output file type. Output format: %s. Output file type: %s.",
 		"outfileMustBeValid": "--outfile must be a well-formed filepath.",
```

Comment:
@jfeingold35 Good catch in the doc that I had the wrong description, wrong row. This is fine AS IS.